.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDatePeriodParser.cs

```diff
@@ -687,11 +687,23 @@ private DateTimeResolutionResult ParseSingleTimePoint(string text, DateObject re
                     ret.Success = true;
                 }
 
-                if (dateContext != null)
+                if (dateContext != null && text != "today")
                 {
                     ret = dateContext.ProcessDateEntityResolution(ret);
                 }
             }
+            else
+            {
+                // Handle "now"
+                var nowPr = ParseNowAsDate(text, referenceDate);
```

Comment:
This is required to properly interpret "now" as date range/period boundary, right? Please also update the code comment accordingly.

Reply:
get it!